js/models/Profile.js

```diff
@@ -31,7 +31,7 @@ export default class extends BaseModel {
     return 'use-sync';
   }
 
-  nested() {
+  get nested() {
```

Comment:
For consistency -- this was the only place in the codebase where `<BaseModel>.nested` is not a getter ( until I reverted the Settings.js slot which you feel breaks address validation. 

In fact, this update was from the previous change set on this branch ( before the merge with the branch where I included the changes that emerged as workable ideas in the discussion ). Previously, @rmisio has said that it should work either way since it is accessed through `_.result`.